src/lessons/08_grid_layout/grid-layout.slides.html

```diff
@@ -0,0 +1,758 @@
+                grid-area: sidebar;
+              }
+              .grid-item-3 {
+                grid-area: footer;
+              }
+
+            # HTML
+            <div class="grid-container">
+              <div class="grid-item-1">Item 1</div>
+              <div class="grid-item-2">Item 2</div>
+              <div class="grid-item-3">Item 3</div>
+            </div>
+    </code>
+                <div class="fragment grid-container grid-10">
+                    <div class="grid-item-1">Item 1</div>
+                    <div class="grid-item-2">Item 2</div>
+                    <div class="grid-item-3">Item 3</div>
+                </div></pre>
+            </section>
+
+            <section>
+                <h3>Grid Line Size</h3>
+                <code>column-gap, row-gap</code>
+                <br>
+                <p><b>Values</b></p>
+                <ul>
+                    <li>&lt;line-size&gt;: size of column/row (px, %, auto, fr)</li>
+                </ul>
+                <p>Only created between tracks!</p>
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+          <code class="formattedCode">
+              # CSS
+                .grid-container {
+                  grid-template-columns: auto auto auto;
+                  column-gap: 10px;
+                  row-gap: 15px;
+                }
+
+              # HTML
+              <div class="grid-container">
+                <div class="grid-item">Item 1</div>
+                <div class="grid-item">Item 2</div>
+                <div class="grid-item">Item 3</div>
+                <div class="grid-item">Item 4</div>
+                <div class="grid-item">Item 5</div>
+              </div>
+      </code><br><div class="fragment grid-container grid-11">
+        <div class="grid-item">Item 1</div>
+        <div class="grid-item">Item 2</div>
+        <div class="grid-item">Item 3</div>
+        <div class="grid-item">Item 4</div>
+        <div class="grid-item">Item 5</div>
+    </div></pre>
+
+            </section>
+
+            <section>
+                <h3>Grid Line Size - Shorthand</h3>
+                <code>gap</code>
+                <br>
+                <p><b>Values</b></p>
+                <ul>
+                    <li>&lt;line-size&gt;: size of column/row (px, %, auto, fr)</li>
+                </ul>
+
+                <pre><code class="formattedCode css">
+                .container {
+                  gap: row-gap column-gap;
+                }</code></pre>
+                <p>If no row-gap is specified, it’s set to value of column-gap</p>
+
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+        <code class="formattedCode">
+            # CSS
+              .grid-container {
+                grid-template-columns: auto auto auto;
+                <!-- column-gap: 10px; -->
+                <!-- row-gap: 15px; -->
```

Comment:
should be css comments

Reply:
[Fixed](https://github.com/ReDI-Munich-Frontend/fed-munich-2021-spring/commit/1c4091a9316975c61acb733fbfdb941f98d3266a)